tools/scripts/CaptureDevHomeLogs.ps1

```diff
@@ -0,0 +1,18 @@
+# Quick script to capture log files and create a zip file for upload to GitHub
+# Currently this only creates one based on Date. We could add a time, so that 
+# the ISV can have more than one Zip file
+
+$MyZip=$(Get-Date -Format yyyy-MM-dd) + "DevHome.Zip"
+$LogPath="C:\Users\" + $Env:UserName  + "\AppData\Local\Packages"
+$LogFiles = Get-ChildItem -Path $LogPath -Filter "*.dhlog" -recurse

```

Comment:
This looks in all packages, recursively, for all .dhlog files?  This might pick up other log files in other packages.

Maybe add *DevHome* to the search.  However, we do specify the devhome versions in / logs.  I think the packages can be hard coded.

Reply:
It might take a while to scan the entire app data folder for .dhlog files, definitely agree the script should have a Dev Home filter. At a minimum, "Microsoft.Windows.DevHome*" should be in the filter path. Iterating over a finite package list (with each release, .Dev, and .Canary variants), could also work, but whether that was worth it would depend on how performant Get-ChildItem is with a more restrictive filter.

Reply:
I tested this and it runs very fast, so I think this is probably fine and would catch any third party extensions that adopt our log format.

Reply:
I also would prefer not to grab everything. This would be run manually after we specifically ask the user to do so. I suggest we change it to only collect Dev Home, with optional parameters for additional packages when we think it's useful.